### DISCUSSION TITLE
Invalid time value when passing date object with params of string month and year

### DISCUSSION THREAD

**Discussion by hownaskie** [js-fns/date-fns#2774] [Q&A]
I got an error in react-native but in debug mode it works fine

```
import { format, subMonths } from 'date-fns';

function getLastMonth(monthYear) { // Jul 2021
    return format(subMonths(new Date(monthYear), 1), 'do MMMM');
}
```

The expected result would be `Jun 2021` but return `Invalid time value`

**Comment by fturmel** [comment]
Have a look at #2141, I think this is the same problem.

Essentially, 
- React Native uses a different js runtime for debugging (V8 vs JavaScriptCore)
- JS runtimes do not parse string inputs in the Date constructor or Date.parse() the same way
